models/cutouts.py

```diff
@@ -433,7 +449,7 @@ def _load_dataset_from_hdf5(self, file, groupname):
         for att in self.get_data_attributes():
             if att == 'source_row':
                 self.source_row = dict(file[f'{groupname}'].attrs)
-            else:
+            elif att in file[f'{groupname}']:
```

Comment:
why use `file[f'{groupname}']` instead of `file[groupname]`? I see above on line 339 of the new file `target = file[groupname].attrs`

Reply:
I don't know why I did it like that. Thanks!